src/wallet/api/wallet2_api.h

```diff
@@ -74,44 +74,50 @@ enum NetworkType : uint8_t {
 */
 struct EnoteDetails
 {
-// from seraphis legacy enote types
     // Ko
-    rct::key onetime_address;
-    /// a
-    rct::xmr_amount amount;
-    /// C
-    rct::key amount_commitment;
-    /// enc(x)
-    rct::key encoded_amount_blinding_factor;
-    /// enc(a)
-    rct::key encoded_amount;
-    /// view_tag
-    crypto::view_tag view_tag;
-
-    // TODO : figure out if we need other members from transfer_details too, these are the ones that are not part of `TransactionInfo`, but my first impression is we could have an overlap so we can find TransactionInfo from EnoteDetails and vice versa.
-// from transfer_details:
-    cryptonote::transaction_prefix m_tx;
-    // index in m_tx.vout
-    uint64_t m_internal_output_index;
-    uint64_t m_global_output_index;
+    std::string m_onetime_address;
+    // view_tag
+    std::string m_view_tag;
+    // QUESTION : I'd argue block height for enote is redundant, you can get this information from TransactionHistory. Or any different opinions?
```

Comment:
I tend to think let's take it in. Probably strictly seen it's redundant, but on the other hand may make a difference when you output lists of enotes in a tool like the CLI wallet, whether you can straight-out just take all the info you need from the returned list, other whether you have to gather and join info first.